code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -90,6 +93,7 @@
 	if(life <= 0)
 		del(src)
 	return
+*/

```

Comment:
hypocritical of me but don't comment out, we have version control if we need to reference it at a later date.